src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableColumns.cs

```diff
@@ -29,8 +29,9 @@ internal static class HotspotsTableColumns
         public static IReadOnlyList<ColumnState> InitialStates { get; } = new[]
         {
             new ColumnState(StandardTableColumnDefinitions.ErrorCode, true, 50),
+            new ColumnState(StandardTableColumnDefinitions.Priority, true, 100),
+            new ColumnState(StandardTableColumnDefinitions.Text, true, 350),
```

Comment:
Minor: changed the order, so that the message/text would appear before the file name